src/System.Net.Http/src/System/Net/Http/Managed/AutoRedirectHandler.cs

```diff
@@ -82,7 +82,7 @@ public AutoRedirectHandler(int maxAutomaticRedirections, HttpMessageHandler inne
                 redirectCount++;
                 if (redirectCount > _maxAutomaticRedirections)
                 {
-                    throw new HttpRequestException("max redirects exceeded");
+                    throw new HttpRequestException(SR.net_http_max_redirects);
```

Comment:
cc: @stephentoub Not specific feedback for this PR, but I noticed that the ManagedHandler is throwing an exception for this.  The current behavior for .NET Core (and we have tests for this) is to simply stop following the redirect and return the 3xx response.